packages/tools/src/tools/displayTools/SegmentationDisplayTool.ts

```diff
@@ -65,13 +64,15 @@ class SegmentationDisplayTool extends BaseTool {
     }
 
     // for each segmentationData, make the visibility false
-    for (const segmentationRepresentation of toolGroupSegmentationRepresentations) {
-      segmentationRepresentation.visibility = true;
-      triggerSegmentationRepresentationModified(
-        toolGroupId,
-        segmentationRepresentation.segmentationRepresentationUID
-      );
-    }
+    toolGroupSegmentationRepresentations.forEach(
```

Comment:
I think this chunk of code is setting the visibility to `true`, yet the comment on line 66 says `false`.